debian/debathena-pyhesiodfs.service

```diff
@@ -0,0 +1,20 @@
+# Author: Lizhou Sha <slz@mit.edu>
+# This file is part of the debathena-pyhesiodfs package.
+
+[Unit]
+Description=Hesiod automounter for Athena lockers
+Requires=openafs-client.service
+After=local-fs.target network.target openafs-client.service
+Before=umount.target
+
+[Service]
+Type=simple
+Environment="pyhesiodfs_dir=/mit"
+OOMScoreAdjust=-1000
+ExecStartPre=/bin/mkdir -p ${pyhesiodfs_dir}
+ExecStartPre=/bin/chown root:pyhesiodfs ${pyhesiodfs_dir}
+ExecStartPre=/bin/chmod 770 ${pyhesiodfs_dir}
+ExecStart=/usr/bin/pyhesiodfs -f ${pyhesiodfs_dir} -o nonempty
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
I am inclined to leave out WantedBy=remote-fs.target, but the logic I use to justify it to myself doesn't really hold water (namely, that we are not a "real" remote filesystem since the data we serve is just symlinks, not actual file contents.  This doesn't really apply, since the contents of the symlinks are the result of (remote) DNS queries.  So I guess you should go ahead and make that change.